EssentialFeed/EssentialFeedTests/Feed API/URLSessionHTTPClientTests.swift

```diff
@@ -92,8 +92,8 @@ class URLSessionHTTPClientTests: XCTestCase {
 		let result = resultFor(values, file: file, line: line)
 
 		switch result {
-		case let .success(data, response):
-			return (data, response)
+		case let .success(successResult):
+			return successResult
```

Comment:
Just a naming suggestion: The name of the function is `resultValuesFor(...)` so I'd expect to return 'values' (instead of `successResult`)

```suggestion
		case let .success(values):
			return values
```